CBLClient/Apps/CBLTestServer-Android/app/build.gradle

```diff
@@ -28,7 +28,7 @@ android {
     }
 }
 
-final String DEF_VERSION = "2.5.0-208"
```

Comment:
no need to checkin this file. you can add it to the gitignore to avoid checking in

Reply:
removed this change.